automatic/picard/picard.nuspec

```diff
@@ -19,14 +27,25 @@ Picard supports the majority of audio file formats, is capable of using audio fi
 When tagging files, Picard uses an album-oriented approach. This approach allows it to utilize the MusicBrainz data as effectively as possible and correctly tag your music. For more information, see the illustrated quick start guide to tagging.
 
 Picard is named after Captain Jean-Luc Picard from the TV series Star Trek: The Next Generation.
+
+## Features
+- Multiple Formats
+  Picard supports all popular music formats, including MP3, FLAC, OGG, M4A, WMA, WAV, and more.
```

Comment:
That is actually not an issue, unlike the initial commonmark spec, chocolatey actually uses hard break lines instead of soft breaks.

If you look for instance of the [waterfox](https://chocolatey.org/packages/waterfox) package, you see there is a line break in after each dot.
However, I have not added two spaces after them since that isn't necessary.
The same thing is happening in the notes section list of that package.